pkgs/top-level/all-packages.nix

```diff
@@ -12326,8 +12318,6 @@ in
 
     org2blog = callPackage ../applications/editors/emacs-modes/org2blog { };
 
-    pcache = callPackage ../applications/editors/emacs-modes/pcache { };
```

Comment:
@mdorman The generated packages only cover packages in `emacsPackagesNg`. They don't do anything about `emacsPackages`, so these packages must stay. Could you make a PR that reverts the removal of `pcache`, `s`, and `xmlRpc`?